### PR TITLE
Small nits for xcm-v3

### PR DESCRIPTION
**Pull request by bkontur**
need this fix to pass: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2132392

### CODE REVIEW

File: xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs [hunk 1]
```diff
@@ -497,11 +497,13 @@ benchmarks! {
 	}
 
 	export_message {
+		let (network, destination) = T::export_message_destination().map_err(|_| BenchmarkError::Skip)?;
```
Comment (acatangiu) [inline xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs:500]: use the error specified by the trait fn implementation:

```suggestion
		let (network, destination) = T::export_message_destination()?;
```

**Review by acatangiu — APPROVED** [review]
Looks good, but shouldn't the new trait fn be impl in `runtime/polkadot/src/lib.rs` too?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2133643

File: xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs (hunk 1 above)
Comment (bkontur) [reply xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs]: benchmark export_message was reverted

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2135701

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2136330

**Comment by bkontur** [comment]
/cmd queue -c bench-bot $ xcm westend-dev pallet_xcm_benchmarks::generic

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2137214

**Comment by bkontur** [comment]
/cmd queue -c bench-bot $ xcm westend-dev pallet_xcm_benchmarks::generic

**Comment by bkontur** [comment]
/cmd queue -c bench-bot $ xcm statemint assets pallet_xcm_benchmarks::generic

**Comment by KiChjang** [comment]
I have completely removed the benchmarks for `ExportMessage`, so this PR would simply just be a weight update.

File: runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs [hunk 2]
```diff
@@ -49,135 +50,138 @@ impl<T: frame_system::Config> WeightInfo<T> {
...
+		Weight::from_ref_time(5_595_000 as u64)
 	}
 	// Storage: XcmPallet Queries (r:1 w:0)
 	pub(crate) fn query_response() -> Weight {
-		Weight::from_ref_time(12_946_000 as u64)
+		Weight::from_ref_time(17_809_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 	}
+	// Storage: Staking Ledger (r:1 w:0)
+	// Storage: Session NextKeys (r:1 w:0)
```
Comment (KiChjang) [inline runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs:69]: This is bad, it's looking into two different pallet's storage just to perform a `Transact`...

Comment (KiChjang) [reply runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs:69]: Okay, jesus, I'm just going to whitelist `remark_with_event` when the `runtime-benchmarks` feature is enabled.

**Comment by bkontur** [comment]
/cmd queue -c bench-bot $ xcm westend-dev pallet_xcm_benchmarks::generic

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2152873

**Review by KiChjang — APPROVED** [review]

**Pull request merged**